eclair-core/src/main/resources/reference.conf

```diff
@@ -36,6 +36,8 @@ eclair {
     }
   }
 
+  min-feerate = 1 // minimum feerate in satoshis per byte
```

Comment:
Do we really need to make this configurable?

Reply:
I think it is safer, otherwise if the minimum relay fee goes up in the bitcoin network we may end up with tx that cannot even be relayed